src/git/config.go

```diff
@@ -35,6 +49,19 @@ type Repository struct {
 	Branches []string `json:"branches"`
 }
 
+// CIAction : Used for unmarshalling the CI request
+type CIAction struct {
+	Repo    string `json:"repo"`
+	Python  string `json:"python"`
+	Golang  string `json:"golang"`
+	Node    string `json:"node"`
+	Ts      string `json:"ts"`
+	Flutter string `json:"flutter"`
+	Dart    string `json:"dart"`
+	Docker  string `json:"docker"`
+	Shell   string `json:"shell"`
```

Comment:
Why do we need all their types to be string. Can't we unmarshall to bool instead from the json request?